main.py

```diff
@@ -118,6 +118,33 @@ def get_incidents():
         )
     }
 
+@app.route("/incidents/<incident_id>", methods=["GET"])
```

Comment:
Do we have any unit tests for each API? If not, how will you ensure your implementation is correct? Most of the bugs can be discovered through well defined unit tests.